### DISCUSSION TITLE
Link that exist show up as red if you don't put a /

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#3697] [Error / Bug Report]
**Describe the bug**
On my home page I have this: `<a href="t">tags page</a>` It shows up as red on the page view. 
Adding a `/` `<a href="/t">tags page</a>`fixes the issue. 

Latest version

**Comment by fireundubh** [comment]
I can confirm this issue. Imported from 1.x and found that some links were red and others were not. The difference between the two was as the reporter described.
